src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataReaderTest/DataReaderTest.cs

```diff
@@ -790,6 +790,95 @@ static char[] Resize(char[] buffer)
oEZHA6NwoEZHA6MwoFZHA6MjgKBWRwOjMyCgVkcDoxMAoFZHA6MjcKBWRwOjMzCgVkcDoyMQoFZHA6MTMKBWRwOjI2CgVkcDozOAoFZHA6MTgKBWRwOjI0CgRkcDo5CgVkcDozMQoFZHA6MTYKBWRwOjExCgVkcDoxOQoFZHA6MjIKBWRwOjIzCgVkcDoyMAoFZHA6MzQKBWRwOjM1CgRkcDo2CgVkcDozNwoEZHA6NQoFZHA6MTUKBWRwOjI1CgRkcDo0CgRkcDoxCgVkcDoxMgoFZHA6MTQKBGk6MjYKBWk6MTY0CgVpOjIyMQoFaToyMjIKBGk6MTQKBWk6MTgxCgNpOjQKBGk6MTAKBGk6MTYKBWk6MTUxCgRpOjE3CgVpOjE0OQoEaTo5NAoEaTo5MwoFaToyMjUKBWk6MjE4CgVpOjIxNgoFaToyNjQKBWk6MjY1CgVpOjI1OQoFaToxODUKBGk6OTEKBGk6OTIKBGk6MjEKBWk6Mjc4CgRpOjcwCgRpOjcxCgRpOjU2CgRpOjU1CgVpOjI4MgoFaToyMDIKBWk6MjA2CgRpOjc0CgRpOjUwCgRpOjgzCgRpOjgyCgRpOjgxCgVpOjI4OQoEaTozNAoFaToyNzIKBWk6MjE3CgVpOjI0NwoFaToyNDYKBWk6MTYxCgRpOjg4CgVpOjIwNwoFaToyMDgKBGk6NDMKBGk6NjYKBWk6MjUzCgVpOjI1NAoEaToxMQoEaToxMgoEaTo5MAoEaTo1MwoEaTozOAoFaToyODEKBWk6MTc5CgVpOjE4OQoFaToyMjYKBWk6MjAxCgRpOjI0CgRpOjI1CgVpOjE1MwoDaToxCgRpOjk2CgRpOjk1CgVpOjEzMQoFaToxMzMKBWk6MTMyCgVpOjE5NQoFaToyNTgKBGk6MzAKBGk6NzcKBGk6MzUKBGk6MjcKBGk6NDkKBGk6MzkKBWk6MjYzCgRpOjQ1CgRpOjQ2CgRpOjQ0CgVpOjI4NQoFaToyODQKBWk6MTQyCgVpOjE0NQoFaToxNTIKBWk6MTY2CgVpOjE2NQoFaToxNTgKBWk6MTYzCgVpOjE0NAoFaToxNDMKBWk6MjczCgVpOjI3NAoFaToyNzUKBWk6MjQyCgVpOjEwMwoFaToxNzIKBWk6MTcxCgVpOjI3MQoFaToxMDAKBWk6MTAxCgRpOjk4CgRpOjk5CgRpOjg3CgVpOjEyOAoFaToxMTAKBWk6MTA5CgVpOjEwNgoFaToxMjMKBWk6MTI3CgVpOjExMgoFaToxMDgKBWk6MTExCgVpOjExNgoFaToxMTUKBWk6MTI5CgVpOjEwNwoFaToxMTkKBWk6MTEzCgVpOjExNAoFaToxMTcKBWk6MTIwCgVpOjExOAoFaToxMjEKBWk6MTIyCgVpOjEyNgoFaToxMjQKBWk6MTI1CgRpOjc1CgVpOjI2OAoFaToyNjkKBWk6MTU5CgVpOjI2NgoFaToyNzAKBWk6MTY4CgRpOjEzCgVpOjE1MAoFaToxNjcKBWk6MjU3CgVpOjIxOQoEaTo2MAoEaTo1OQoFaToxMzcKBWk6MTM4CgVpOjIyNwoDaTo5CgVpOjI0MwoEaTo2NQoFaToyMjMKBWk6MjI0CgRpOjY4CgRpOjY0CgVpOjE2MAoFaToyMDMKBGk6ODAKBWk6MTk3CgVpOjE4MgoEaToyOAoEaTo3OQoEaTo4NQoFaToyNjcKBGk6NTEKBGk6NzMKBGk6NzgKBGk6ODQKBGk6ODkKBWk6MTM0CgVpOjE3MwoFaToxMDQKA2k6NwoEaToyMgoEaTo1MgoFaToxNDYKBWk6MTk0CgRpOjM2CgRpOjQwCgVpOjE2MgoFaToyNDUKBWk6MTk5CgRpOjY5CgRpOjY3CgVpOjE1NAoFaToxNzYKBWk6MjIwCgVpOjIzMwoFaToxNTYKBWk6MTkzCgRpOjg2CgVpOjE5OAoFaToyNzcKBWk6MjA5CgVpOjIxMAoFaToyMzIKBWk6MjE1CgRpOjYyCgRpOjMyCgVpOjE3MAoFaToxNTcKBWk6MjUyCgRpOjU0CgRpOjQxCgVpOjE1NQoFaToyNzkKBWk6MTY5CgVpOjE0NwoFaToxNDgKBGk6MzcKBGk6MjkKBWk6MTc4CgVpOjE3NwoFaToyMTMKBGk6MzMKBWk6MjgzCgVpOjE3NQoEaTo1OAoEaToyMAoEaTo2MwoEaToxNQoDaTozCgVpOjIwMAoDaToyCgVpOjI2MQoFaToyODYKBWk6MTM1CgVpOjEzNgoFaToyNzYKBGk6NDcKA2k6OAoEaTo5NwoFaToyODgKBWk6MTM5CgVpOjI4NwoFaToxOTAKBWk6MTkyCgVpOjE5MQoEaToxOAoEaTo3NgoFaToxMzAKBWk6MjA0CgVpOjI2MAoFaToyMzEKBWk6MjE0CgRpOjYxCgVpOjE0MQoFaToyODAKBWk6MjI4CgVpOjIyOQoFaToyMzQKBWk6MjM1CgVpOjI1MAoFaToyNTEKBWk6MTg2CgVpOjE4OAoFaToxODcKBWk6MjMwCgRpOjQyCgVpOjE5NgoFaToyNDAKBWk6MjM5CgVpOjE0MAoEaToyMwoFaToyNDEKBWk6MTc0CgRpOjcyCgVpOjIwNQoFaToyNTYKBWk6MjU1CgRpOjE5CgRpOjQ4CgNpOjYKBWk6MTAyCgVpOjIxMQoFaToyMTIKBGk6NTcKBWk6MjYyCgVpOjI0NAoFaToyMzYKBWk6MjM3CgVpOjIzOAoDaTo1CgVpOjEwNQoFaToyNDgKBWk6MjQ5CgVpOjE4MwoFaToxODQKBGk6MzEKBWk6MTgwCgN0OjMKBHQ6MTEKBHQ6MTAKBHQ6MjEKA3Q6NAoDdDo2CgN0OjkKA3Q6NwoDdDo4CgR0OjEzCgR0OjE4CgR0OjEyCgR0OjE5CgR0OjE3CgN0OjUKBHQ6MTUKA3Q6MQoEdDoyMAoDdDoyCgR0OjE2CgR0OjE0";
```

Comment:
Why not generate it... we have lots of code that generates random values, could be a good opportunity to add a utility to common

Reply:
Because i know this specific value has the required characteristics when split over multiple packets to cause the original problem 100% effectively. I could spend time trying to reduce it but it gains me nothing.